Lib/unittest/case.py

```diff
@@ -86,23 +86,10 @@ def _id(obj):
 
 
 _module_cleanups = []
-def addModuleCleanup(*args, **kwargs):
+def addModuleCleanup(function, /, *args, **kwargs):
```

Comment:
Can `addCleanup` and `addClassCleanup` also be refactored? Similar to `addModuleCleanup` there is a test case failure in `test_addCleanup_arg_errors` that needs to be fixed.

Reply:
`addModuleCleanup` was added in 3.8, so we can change it. `addCleanup` existed before, so it needs a deprecation period. Thank you for pointing on `addClassCleanup`, I forgot about it.

Reply:
`addCleanup` seems to have a deprecation message already and the message could be modified to indicate removal of it in future version like 3.9 to make `function` positional only argument.

Reply:
Yes, most of deprecations was added in bpo-36492.